### ISSUE TITLE
8.2.2 Missing

### ISSUE THREAD

**Issue by anpieber** [cptactionhank/docker-atlassian-jira-software#66]
https://my.atlassian.com/download/feeds/current/jira-software.json is pointing to 8.2.2 but there is still no tag created for it https://hub.docker.com/r/cptactionhank/atlassian-jira-software any ideas what's going wrong here?

Thank you very much and kind regards,
Andreas

**Comment by smurfy** [comment]
see #61 too.

i've checked circleCI logs at:

https://circleci.com/gh/cptactionhank/docker-atlassian-jira-software/3076

While commiting the 8.2.2 changes to master as new changes, it seems that the check for remote branch existance failes somehow on circleCI (works for me on my linux box)

``
Checking if branch is already available?
Branch '8.2.2' already exist. Skipping...
``

**Comment by vrobert78** [comment]
Hi,

the code in the latest tag is well in 8.2.2.
It's seems that just the tag 8.2.2 is missing.

**Comment by smurfy** [comment]
yes, the docker hub, tag is created via the branch within the git repo. Since there is some kind of bug with the circleCI, it failes to create the branch, because it thinks it already exist.

**Comment by anpieber** [comment]
The script works perfectly on my local box too. Next steps might be to clear the circleCI cache (is there even one?) and/or increase the log density in the build script. Is there anything I can help with? I really like the image and would definitely prefer not to use "latest" :-)

**Comment by zacheryph** [comment]
I think? this commit https://github.com/cptactionhank/docker-atlassian-jira-software/commit/f511df1e6334f9f6b02393013f851ec8a7535cc4 caused the issue? the confluence image still has the old code and its still tagging properly. But it doesn't really explain how the `8.1.0` tag got to Github/Docker short of it being tagged manually? It is attached to the latest _8.1.0 update_ on master. None of the newer 7.13 past `7.13.1` exist on Github/Docker either.

**Comment by loneken79** [comment]
As a workaround you can pull the latest images and tag it localy, so you can use it in your process.

Assuming the latest commit is refering to the desired tag update (8.3.0 actually) the commands are:
1. `docker pull cptactionhank/atlassian-jira-software` (get "latest" version of the image)
2. `docker tag cptactionhank/atlassian-jira:latest cptactionhank/atlassian-jira:8.3.0` (tag this image locally)
3. check that you see both "latest" and "8.3.0" version with the same "Image ID" in `docker images | head` listing
4. `docker rmi cptactionhank/atlassian-jira:latest` (clean unused "latest" version)

**Comment by viceice** [comment]
There is now an official docker image [jira-software](https://hub.docker.com/r/atlassian/jira-software)
